x-pack/lib/filters/geoip/database_manager.rb

```diff
@@ -102,7 +101,7 @@ def execute_download_job
       pipeline_id = ThreadContext.get("pipeline.id")
       ThreadContext.put("pipeline.id", nil)
 
-      update_download_status(:updating)
+      @database_metric.set_download_status_updating
```

Comment:
the more I see references to the ivar `@database_metric`, the more I think that we should instead reference an instance method that supplies a null metric implementation.